example_files/example_project_combined.json

```diff
@@ -675,7 +675,7 @@
         "number_of_bedrooms": 4,
         "foundation_type": "basement - unconditioned",
         "attic_type": "attic - unvented",
-        "system_type": "Residential - boiler and room air conditioner",
+        "system_type": "Residential - furnace and central air conditioner",
```

Comment:
Why change all of these to "furnace and central ac"? They were intentionally different so that we test other combinations.

Reply:
Thanks for catching that, I will revert this.